commit faaa943aacaeaee27e5a309bb54bd539ec1a58a2
Author: Drup <drupyog@zoho.com>
Date:   Sat Dec 6 05:08:13 2014 +0100

    Replace use of the deprecated `Lwt_event` by `Lwt_react.E`.

diff --git a/src/extensions/ocsigen_comet.ml b/src/extensions/ocsigen_comet.ml
index ad7d9dd..ccc0719 100644
--- a/src/extensions/ocsigen_comet.ml
+++ b/src/extensions/ocsigen_comet.ml
@@ -468,7 +468,7 @@ end = struct
            Lwt.choose
              [ (choosed >|= fun x -> Some x);
                (Lwt_unix.sleep (get_timeout ()) >|= fun () -> None);
-               (Lwt_event.next Security.kill >>= fun () -> Lwt.fail Kill);
+               (Lwt_react.E.next Security.kill >>= fun () -> Lwt.fail Kill);
              ] >|= fun x ->
            List.iter (fun c -> Channels.send_listeners c (-1)) active ;
            let s = Messages.encode_downgoing ended x in
